examples/configuration/configure-logging/src/main/java/com/hazelcast/jet/examples/configuration/logging/JetInstance.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.jet.examples.configuration.logging;
+
+import com.hazelcast.jet.Jet;
+
+public class JetInstance {
+    public static void main(String[] args) {
+        Jet.newJetInstance();
```

Comment:
isn't it better to just set mainclass to StartServer? this seems redundant..

Reply:
Since StartServer initializes JDK logging by default it is better to keep it.